### ISSUE TITLE
This addon breaks bluetooth on my Pi

### ISSUE THREAD

**Issue by shabbir-genetech** [jvcleave/ofxRPiCameraVideoGrabber#26]
I kept facing an issue with an app that I created which used this addon: I would run my app (which had nothing to do with bluetooth itself (at that point)) and once my app exited, I would be unable to use any bluetooth functionality on my Pi. `hcitool scan`, `bluetoothctl` and `ussp-push` would all show timeouts and `sudo btmon` would not capture any traffic until I restarted the Pi.

After 2 weeks trying to locate the culprit, I finally found this addon to be the cause. Removing it from my app fixed the issue. I ended up replacing it with https://github.com/orgicus/ofxCvPiCam. Just wanted to put this out here in case someone else is facing the issue or someone is interested in exploring and fixing the issue.

P.S.: This issue occurs with the addon's examples too. I just make and run https://github.com/jvcleave/ofxRPiCameraVideoGrabber/tree/master/example-texture-mode and it breaks bluetooth for me.

**Comment by jvcleave** [comment]
interesting - do you ever see a rainbow box overlay in the upper right corner? (sounds like a power related issue)

**Comment by shabbir-genetech** [comment]
Yes, sometimes I do see that. However, I replacing this addon with the other one did fix the issue for me while I still get the rainbow box overlay.

**Comment by jvcleave** [comment]
ofxCvPiCam works a bit differently as it doesn't use the egl_render component so it likely uses less power. 

Power issues can cause weird stuff to happen. I have seen ethernet drop, HDMI quit working, the strangest one was where I could ssh in but it wouldn't accept my password. I would try a better power supply (esp with a RPi3). This is the one I just switched to:

https://www.amazon.com/gp/product/B00MARDJZ4

**Comment by shabbir-genetech** [comment]
Changed the power supply. Not getting the rainbow box overlay anymore. Can still reproduce the issue.
